lib/child_process.js

```diff
@@ -109,7 +97,6 @@ function normalizeExecArgs(command /*, options, callback*/) {
 exports.exec = function(command /*, options, callback*/) {
   var opts = normalizeExecArgs.apply(null, arguments);
   return exports.execFile(opts.file,
-                          opts.args,
```

Comment:
Prior to this change, `args` would have been something like `['-c', command]`. That would have been sent through `execFile()`, `spawn()`, etc.

With this change, `normalizeExecArgs()` sets `shell` in options instead. This is passed through until `normalizeSpawnArguments()`, where the same `['-c', command]` is created and passed to `child.spawn()`.